### ISSUE TITLE
file ingest crashes if same file added twice

### ISSUE THREAD

**Issue by gjost** [densho/ddr-cmdln#38]
File ingest fails if user tries to add files twice.
Celery error message:
```
Could not upload ddr-densho-296-98-1_mezz.tif to ddr-densho-296-98.
Exception('Add file aborted, see log file for details: /var/log/ddr/addfile/...
```
Collection repo is left in an inconsistent state.

UPDATE: Modify file ingest to look for file at destination path once it has a `sha1`, and quit/die if the file already exists in the repo.  This will prevent modifications.

**Comment by gjost** [comment]
[20160830-ddr-densho-296_git-status.txt](https://github.com/densho/ddr-cmdln/files/445480/20160830-ddr-densho-296_git-status.txt)

[ddr-densho-296-98AddFileError.log.txt](https://github.com/densho/ddr-cmdln/files/445481/ddr-densho-296-98AddFileError.log.txt)

**Comment by gjost** [comment]
`ddr-cmdln` on the problem machine is on `master` at `eb91a0e283`.

**Comment by gjost** [comment]
Fixed in 778071d659.
